lib/dor/text_extraction/speech_to_text_cocina_updater.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Dor
+  module TextExtraction
+    # Update Cocina structural metadata with speech to text files files
+    class SpeechToTextCocinaUpdater < CocinaUpdater
+      private
+
+      # set the use attribute based on the mimetype
+      def use(object_file)
+        case object_file.mimetype
+        when 'text/plain'
+          'transcription'
+        when 'text/vtt'
+          'caption'
+        end
+      end
```

Comment:
Do we care about when the mimetype doesn't match?

Reply:
yes, it will be nil in that case, which results in a blank role, which is what we want

if we have other mimetypes later than need specific roles, they can be easily added here